submissions/AsyaYeromina/Tiny-JS-world/index.js

```diff
@@ -0,0 +1,93 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/AsyaYeromina/a-tiny-JS-world
+   Web app: https://asyayeromina.github.io/a-tiny-JS-world/.
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const dog = {
+   species: 'Canis',
+   name: 'Jessie',
+   gender: 'female',
+   legs: '4',
+   hands: '0',
+   speech: 'woof-woof!',
+   friends: ['Stepan', 'Ivanka']
+};
+
+const cat = {
+   species: 'Felis catus',
+   name: 'Python',
+   gender: 'male',
+   legs: '4',
+   hands: '0',
+   speech: 'Meow',
+   friends: []
+};
+
+const woman = {
+   species: 'Homo sapiens',
+   name: 'Ivanka',
+   gender: 'female',
+   legs: '2',
+   hands: '2',
+   speech: 'Hello!',
+   friends: ['Stepan', 'Python', 'Jessie']
+};
+
+const man = {
+   species: 'Homo sapiens',
+   name: 'Stepan',
+   gender: 'male',
+   legs: '2',
+   hands: '2',
+   speech: 'Hello!',
+   friends: ['Ivanka', 'Python', 'Jessie']
+};
+
+const catWoman = {
+   species: 'Felis sapiens',
+   name: 'Selina',
+   gender: 'female',
+   legs: '2',
+   hands: '2',
+   speech: cat.speech,
+   friends: ['Stepan', 'Python']
+};
+
+const inhabitants = [cat, dog, woman, man, catWoman];
+const keys = ['species', 'name', 'gender', 'legs', 'hands', 'speech', 'friends'];
+
+const getValues = function(obj, arr) {
```

Comment:
`getValues` is too generic.
What this function does? It composes an inhabitant's individual properties into the inhabitant's presentation. Something like `getInhabitantPresentation` or `build...` or `buildPresentation`.

`obj` and `array` are too generic.
What the function expects as its 1st argument? It is specialized to process inhabitants. So `inhabitant` tells a reader what the function expects here.

Not just any `array` is ok here but rather `propertyNames`.

This will also make use of IDE IntelliSense, when you start typing a function call and Intellisense gives you a hint on what are the function's arguments names are, so you know what params to pass and in what order. Imagine, if you had `function(arr1, arr2){...}` IntelliSense would give any sensible hint.